website/docs/reference/deprecations.md

```diff
@@ -174,6 +213,26 @@ The following are recommended approaches:
 1. Replace `dbt_modules` with `dbt_packages` in your `clean-targets` spec (and `.gitignore`).
 2. Set `packages-install-path: dbt_modules` if you want to keep having packages installed in `dbt_modules`.
 
+### PackageMaterializationOverrideDeprecation
+
+The behavior where installed packages could override built-in materializations without your explicit opt-in is deprecated. Setting the [`require_explicit_package_overrides_for_builtin_materializations` flag](/reference/global-configs/behavior-changes#package-override-for-built-in-materialization) to `false` in your `dbt_project.yml` allowed packages that matched the name of a built-in materialization to continue to be included in the search and resolution order.
+
+#### PackageMaterializationOverrideDeprecation warning resolution
+
+Explicitly override built-in materializations, in favor of a materialization defined in a package, by reimplementing the built-in materialization in your root project and wrapping the package implementation.
+
+For example: 
```

Comment:
Can we be more explicit? I'm not sure if this is correct @graciegoheen. Also do you add it anywhere or is there a particular place?
```suggestion
For example, add this to your snowflake package:
```

Reply:
it's in your root project

Reply:
https://docs.getdbt.com/reference/global-configs/behavior-changes#package-override-for-built-in-materialization

Reply:
Cited above